python/graphstorm/gconstruct/utils.py

```diff
@@ -29,6 +29,7 @@
 
 import numpy as np
 import dgl
+from dgl import distributed as dgl_distributed
```

Comment:
Will the import script require to use dgl version >= 2.1.0+?

Reply:
Not at this point. The `distributed` module has been part of DGL since v1. Further down we check for the correct version.